### PR TITLE
Update picosheet.red

### PR DESCRIPTION
**Pull request by eranws**
fix: yellow color now clears on un-focus

### CODE REVIEW

File: Showcase/picosheet.red [hunk 1]
```diff
@@ -49,7 +49,7 @@ repeat y 9 [
 			]
 			actors: context [
 				on-create: on-unfocus: function [f e][
-					f/color: none
+					f/color: white
```
Comment (dockimbel) [inline Showcase/picosheet.red:52]: What problem are you trying to fix there? The yellow color already disappears in cells on unfocusing.

**Comment by eranws** [comment]
Sorry, I've used red 0.6.2 by mistake.
It does clear the color on 0.6.3.

**Comment by dockimbel** [comment]
Thanks. Though the typo fix was good, you should resubmit it.

**Comment by eranws** [comment]
Thank you, see fix in #44

**Pull request closed without merging**